hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -98,10 +97,12 @@ public RDBStore(File dbFile, ManagedDBOptions dbOptions,
 
     try {
       if (enableCompactionDag) {
-        rocksDBCheckpointDiffer = RocksDBCheckpointDifferHolder.getInstance(
+        rocksDBCheckpointDiffer = new RocksDBCheckpointDiffer(
```

Comment:
With this, `RocksDBCheckpointDifferHolder` is no longer used right?

Reply:
RocksDBCheckpointDifferHolder can be removed

Reply:
Yes, I forgot to remove that. Just removed. Thanks for pointing it out.